backend-lib/src/main/rust/lib.rs

```diff
@@ -997,21 +994,20 @@ pub extern "C" fn Java_cash_z_ecc_android_sdk_internal_jni_RustBackend_getTotalT
 
         let min_confirmations = NonZeroU32::MIN;
 
-        let amount = db_data
+        let (target, _) = db_data
             .get_target_and_anchor_heights(min_confirmations)
-            .map_err(|e| anyhow!("Error while fetching target height: {}", e))
-            .and_then(|opt_target| {
-                opt_target
-                    .map(|(target, _)| target)
-                    .ok_or(anyhow!("Target height not available; scan required."))
-            })
-            .and_then(|target| {
-                db_data
-                    .get_spendable_transparent_outputs(&taddr, target, 0)
-                    .map_err(|e| anyhow!("Error while fetching verified balance: {}", e))
-            })?
+            .map_err(|e| anyhow!("Error while fetching target height: {}", e))?
+            .context("Target height not available; scan required.")?;
+
+        let amount = db_data
+            .get_spendable_transparent_outputs(
+                &taddr,
+                target,
+                wallet::ConfirmationsPolicy::new_symmetrical(NonZeroU32::MIN, true),
```

Comment:
Non-blocking: as of latest librustzcash, this can be:
```suggestion
                wallet::ConfirmationsPolicy::MIN,
```